src/EventStore.Core/ClusterVNodeStartup.cs

```diff
@@ -264,13 +264,15 @@ public class ClusterVNodeStartup<TStreamId> : IStartup, IHandle<SystemMessage.Sy
 
 		private class StatusCheck {
 			private readonly ClusterVNodeStartup<TStreamId> _startup;
+			private readonly int _livecode;
```

Comment:
Minor thing, but you can instantiate this here
```suggestion
			private readonly int _liveCode = 204;
```

Reply:
Done